screens/request_profile.js

```diff
@@ -0,0 +1,207 @@
+import React from 'react';
+import { StyleSheet, Text, View, Dimensions, Image, FlatList} from 'react-native';
+
+const window = Dimensions.get("window")
+
+let colors = ['#6DC4E0', '#605DF1']
+
+let items = [
```

Comment:
This is a good array - just change the following things:

* Move the first item out of this list and just render outside of the `FlatList` since it's not a store item
* Change the key from `q` to `quantity` so it's easy to read and understand